### ISSUE TITLE
Description for SwiftUI toggles missing accessibility value

### ISSUE THREAD

**Issue by DavidBrunow** [cashapp/AccessibilitySnapshot#181]
I'd expect the description to include ", off" at the end to describe the toggle's description or ", on" if the toggle were on to match the description from VoiceOver.

![Output from accessibility snapshot testing tool with the VoiceOver description for a SwiftUI toggle. The description generated does not include the value of "off"](https://github.com/cashapp/AccessibilitySnapshot/assets/1452879/0effd2b1-0b3a-47d1-b5a7-a8a78eb3e902)

**Comment by DavidBrunow** [comment]
Please ignore my noise, my local copy of the repo was not up-to-date with the fix made in November.

**Comment by DavidBrunow** [comment]
That being said, we could use a release with the fix.
